packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -0,0 +1,283 @@
+import _ from 'lodash'
+
+import {
+  RouteHandler,
+  RouteMatcherOptions,
+  RouteMatcher,
+  StaticResponse,
+  HttpRequestInterceptor,
+  STRING_MATCHER_FIELDS,
+  DICT_STRING_MATCHER_FIELDS,
+  AnnotatedRouteMatcherOptions,
+  AnnotatedStringMatcher,
+  NetEventFrames,
+  StringMatcher,
+  NumberMatcher,
+} from '@packages/net-stubbing/lib/types'
+import {
+  validateStaticResponse,
+  getBackendStaticResponse,
+  hasStaticResponseKeys,
+} from './static-response-utils'
+import { registerEvents } from './events'
+import $errUtils from '../../cypress/error_utils'
+
+/**
+ * Get all STRING_MATCHER_FIELDS paths plus any extra fields the user has added within
+ * DICT_STRING_MATCHER_FIELDS objects
+ */
+function getAllStringMatcherFields (options: RouteMatcherOptions): string[] {
+  return STRING_MATCHER_FIELDS
+  .concat(
+    // add the nested DictStringMatcher values to the list of fields to annotate
+    _.flatten(
+      _.filter(
```

Comment:
the _.filter without a callback is removing falsy values

Reply:
Ah I didn't know it had that behavior. I think [_.compact](https://lodash.com/docs/4.17.15#compact) would be better since it's clearer that's its purpose.